File: WebScraper/model/Product.py
class Product:
    def __init__(self, brand, model, price, link, product_type, img_src):
        self.brand = brand
        self.model = model
        self.price = price
        self.link = link
        self.product_type = product_type
        self.img_src = img_src

    def to_string(self):
        print(f'Brand: {self.brand}')
        print(f'Model: {self.model}')
        print(f'Price: {self.price}')
        print(f'Link: {self.link}')
        print(f'Type: {self.product_type}')
        print(f'Image src: {self.img_src}')

File: Backend/src/main/java/com/ftn/thesisProject/model/Product.java
package com.ftn.thesisProject.model;

import com.ftn.thesisProject.model.enumerations.ProductType;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class Product {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String brand;
    private String model;
    private Double price;
    private String link;
    private ProductType type;

    public void setId(Long id) {
        this.id = id;
    }

    public Long getId() {
        return id;
    }


    public String getBrand() {
        return brand;
    }

    public void setBrand(String brand) {
        this.brand = brand;
    }

    public String getModel() {
        return model;
    }

    public void setModel(String model) {
        this.model = model;
    }

    public Double getPrice() {
        return price;
    }

    public void setPrice(Double price) {
        this.price = price;
    }

    public String getLink() {
        return link;
    }

    public void setLink(String link) {
        this.link = link;
    }

    public ProductType getType() {
        return type;
    }

    public void setType(ProductType productType) {
        this.type = productType;
    }

}
